src/praisonai-agents/praisonaiagents/process/process.py

```
@@ -348,6 +348,43 @@ def _create_loop_subtasks(self, loop_task: Task, decision_mode: bool = False):
+
+        Combines the task's structured result (json_dict / pydantic fields) with
+        the raw output so expressions like ``{{score}} > 80`` can resolve, and
+        exposes the raw text as ``previous_output``.
+        """
+        context: Dict[str, Any] = {}
+        result = getattr(current_task, "result", None)
+        if result is not None:
+            try:
+                context.update(result.to_dict())
+            except Exception:
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)